README.md

```diff
@@ -21,33 +60,47 @@ $ bower install gaia-components/gaia-button
 Add `disabled` attribute to disable the button.
 
 ```html
-<gaia-button disabled>button disabled</gaia-button>
+<gaia-button disabled><span data-l10n-id="disable">button disabled</span></gaia-button>
 ```
 
-### With icon
+### Circular
 
-Button Can show with icon (need import `gaia-icons`) at left
+Add `circular` attribute to turn button into circular form.
 
 ```html
-<gaia-button><i data-icon="back-light" aria-hidden="true"></i><span>Back</span></gaia-button>
+<gaia-button circular>
+  <span data-icon="camera" data-l10n-id="camera"/>
+</gaia-button>
 ```
 
-or right
 
-```html
-<gaia-button><span>Forward</span><i data-icon="forward-light" aria-hidden="true"></i></gaia-button>
-```
+## RTL
 
-Remember to add `aria-hidden="true"` when both icon and text existed for **Accessibility**. So the screen reader will read the button text and ignore the icon text.
+`gaia-button` is RTL ready. The icon and text position will be changed base on RTL settings.
 
-### Circular
 
-Add `circular` attribute to turn button into circular form.
+## Accessibility
+
+`aria-disabled="true"` is added automatically when button comes with `disabled` attribute. So the screen reader will read the button text and note it was disabled.
+
+* standalone icon butotn:
 
 ```html
-<gaia-button circular><i data-icon="camera"></i></gaia-button>
+<gaia-button>
+  <span data-l10n-id=""/>
```

Comment:
data-l10n-id="" can be on the gaia-button itself in general.
It depends on what we are trying to achieve with l10n.
* if icon[s] is irrelevant to content:
  * we can use aria-hidden on the element that has data-icon iff it is inside the button (e.g. span). aria-hidden will affect only that element.
  * we can apply aria-label (via data-l10n-id) onto the gaia-button itself that describes the button entirely. Since the icon is irrelevant, the label value will be identical to the textual content of the button. Note in this case span is not needed.

* if Icon is relevant to content:
  * if data-icon is on inner span, we must provide data-l10n-id for the span and ensure that the value for aria-label + optional text content make sense
  * if data-icon is on the gaia-button itself we must provide data-l10n-id for the gaia-button describing the button the same way as it would be perceived by the sighted user i.e. icon meaning + inner text content